packages/eslint-plugin/src/rules/shorthand-property-sorting/__tests__/rule.test.ts

```diff
@@ -682,28 +743,23 @@ tester.run('shorthand-property-sorting', shorthandFirst, {
         import { Box } from '@atlaskit/primitives';
 
         const styles = cssMap({
-          root: { paddingTop: '...' },
           success: {
             padding: '...',
           },
+          root: { paddingTop: '...' },
```

Comment:
moved `paddingTop` to the bottom, so that we can verify that the ESLint violations are caused by the `cx(styles.root, styles.success)` function call, which applies `paddingTop` before `padding`